### DISCUSSION TITLE
How do I set the motherboard colors on PC startup/boot?

### DISCUSSION THREAD

**Discussion by pritster5** [jath03/openrgb-python#56] [Q&A]
OpenRGB.exe works great for me except for one problem: It doesn't persist any changes through PC restarts and there's no way to automatically run it on startup as far as I know. 

I would like to use this SDK to write a python script to set the color of all my motherboard LED's to white on PC startup. How would I automate this? 

Also, if there's a way to do this using regular old OpenRGB.exe, please let me know.

**Comment by jath03** [comment]
Look for the "Start At Login" settings in the general settings tab of OpenRGB.  It should be there if you're using a decently new version.

If that doesn't work out for whatever reason, there's also this: https://gitlab.com/CalcProgrammer1/OpenRGB/-/wikis/Frequently-Asked-Questions#can-i-have-openrgb-start-automatically-when-i-log-in

**Reply by pritster5** [reply]
Ok so Turns out I was on an older version. After updating, I can indeed enable "Start at Login".

However, there seems to be an issue as it doesn't actually work. When I hibernate my PC and log back in, OpenRGB does nothing. It doesn't start, and it doesn't load the selected profile.

These are my settings:
![image](https://user-images.githubusercontent.com/7132319/160948062-79729e31-489e-4323-ada6-17a0b4cbb207.png)
